python/_fio_plugin/README.md

```diff
@@ -0,0 +1,44 @@
+# arca-fio
+
+
+
+## Run the thing
+
+### Containerized
+
+#### Requirements
+
+* fio_plugin image named `fio`
+
+```shell
+podman run --volume $(pwd)/poisson-rate-submission_input.yaml:/plugin/fio-job.yaml:z fio
```

Comment:
Think upstream. While I prefer to default to `podman` as well, we should probably document things using the `docker` command instead.

Reply:
Also, when when [SDK PR #36](https://github.com/arcalot/arcaflow-plugin-sdk-python/pull/36) is implemented, we can simplify this by passing stdin to the container and thus not requiring the volume mount.